mwdb/web/src/components/RichAttribute/MarkedMustache.tsx

```diff
@@ -212,8 +226,43 @@ class MarkedTokenizer extends Tokenizer {
 const mustacheWriter = new MustacheWriter();
 const markedTokenizer = new MarkedTokenizer();
 
+type stringOpFunc = (I: string) => string;
+
+const lambda = (func: stringOpFunc = _.identity) => {
+    // A factory method for custom lambdas.
+    //
+    // The inner method receives the text inside the section
+    // and the subrenderer. It then renders the value, and passes
+    // it to a user defined function.
+    //
+    // E.g.: ("{{name}}", renderer) => "John Doe" 
+    // (func is toUpperCase)        => "JOHN DOE"
+    return () => function(text: string, renderer: any): string {
+        return func(renderer(text.trim()));
+    }
+};
+
+const lambdas = fromPlugins("mustacheExtensions").reduce(
+    (prev, curr) => {
+        return { 
+            ...prev, 
+            ..._.mapValues(curr, (func: stringOpFunc) => lambda(func)) 
+        }
+    },
+{});
+
```

Comment:
Global scope is too early for call to `fromPlugins` as the list of plugin extensions is loaded lazily during initial render (https://github.com/CERT-Polska/mwdb-core/blob/master/mwdb/web/src/index.jsx#L19)

I think it should be moved to the renderValue scope.